src/Provider/Provider.Web/Orchestrators/Part1/LocationOrchestrator.cs

```diff
@@ -41,44 +43,55 @@ public class LocationOrchestrator : EntityValidatingOrchestrator<Vacancy, Locati
             var vacancy = await Utility.GetAuthorisedVacancyForEditAsync(
                 _providerVacancyClient, _recruitVacancyClient, vrm, RouteNames.Location_Get);
 
-            return await GetViewModelAsync(vacancy, employerInfoModel?.HasLegalEntityChanged, employerInfoModel?.LegalEntityId);
-        }
-        private async Task<LocationViewModel> GetViewModelAsync(Vacancy vacancy, bool? hasLegalEntityChanged, long? selectedOrganisationId)        
-        {
-            var employerData = await _providerVacancyClient.GetProviderEmployerVacancyDataAsync(vacancy.TrainingProvider.Ukprn.GetValueOrDefault(), vacancy.EmployerAccountId);
+            var vm = new LocationViewModel();
+            vm.PageInfo = Utility.GetPartOnePageInfo(vacancy);
+           
+            long? selectedOrganisationId = employerInfoModel?.LegalEntityId;
```

Comment:
Can we copy the code from Employer.Web LocationOrchestrator? 
`var legalEntityId = employerInfoModel?.LegalEntityId != null ? employerInfoModel.LegalEntityId : vacancy.LegalEntityId;`

`selectedOrganisationId` 
 adds confusion.  We should standardise the code as much as possible between the applications.